addons/l10n_br/__manifest__.py

```diff
@@ -47,14 +47,18 @@
         'account',
```

Comment:
Maybe we can update the manifest description as well?  (remove the launchpad stuff...)

Reply:
Done, with thanks to @ren-odoo :)